### PR TITLE
Fix all known bugs in LoadGoogleSpreadsheetsToSnowflakeWorkflow

### PR DESCRIPTION
**Pull request by pwnage101**
DE-1919

---

edx-analytics-pipeline: This PR
analytics-secure: https://github.com/edx-ops/analytics-secure/pull/327
analytics-secure x2: https://github.com/edx-ops/analytics-secure/pull/328
jenkins-job-dsl: https://github.com/edx/jenkins-job-dsl/pull/875

### CODE REVIEW

File: edx/analytics/tasks/warehouse/load_google_sheet_to_warehouse.py [hunk 1]
```diff
@@ -247,6 +259,7 @@ def requires(self):
         gs = create_google_spreadsheet_client(credentials_target)
         for spreadsheet_key, config in self.spreadsheets_config.items():
             schema = config['schema']
+            scratch_schema = config['scratch_schema']
```
Comment (brianhw) [inline edx/analytics/tasks/warehouse/load_google_sheet_to_warehouse.py:269]: Where is this being used?  Shouldn't it be passed below as a Snowflake-related param?

Comment (pwnage101) [reply edx/analytics/tasks/warehouse/load_google_sheet_to_warehouse.py:269]: yep, forgot to pass it along.

**Review by doctoryes — APPROVED** [review]

**Review by bmedx — APPROVED** [review]

**Review by brianhw — APPROVED** [review]
LGTM.   👍

File: edx/analytics/tasks/warehouse/load_google_sheet_to_warehouse.py [hunk 2]
```diff
@@ -146,6 +146,15 @@ def insert_source_task(self):
             overwrite=self.overwrite,
         )
 
+    @property
+    def null_marker(self):
+        """
+        Empty cells in Google Spreadsheet become simply sequences in the API
+        response.  If we treat the empty string as the NULL sequence, then they
```
Comment (doctoryes) [inline edx/analytics/tasks/warehouse/load_google_sheet_to_warehouse.py]: Nit: `Treat the empty string as the NULL marker, so empty strings will become NULL in Snowflake.`

**Review by doctoryes — APPROVED** [review]

**Pull request merged**